x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/predicate/operator/arithmetic/AddTests.java

```diff
@@ -99,7 +99,8 @@ public static Iterable<Object[]> parameters() {
                     new TestCaseSupplier.TypedData(lhs, DataTypes.DATETIME, "lhs"),
                     new TestCaseSupplier.TypedData(rhs, EsqlDataTypes.DATE_PERIOD, "rhs")
                 ),
-                "AddDatetimesEvaluator[lhs=Attribute[channel=0], rhs=Attribute[channel=1]]",
+                // TODO: There is an evaluator for Datetime + Period, so it should be tested. Similarly below.
+                "No evaluator, the tests only trigger the folding code since Period is not representable",
```

Comment:
@not-napoleon : This is a situation that I mentioned before, where it looks like we're testing the evaluator but we don't.